### PR TITLE
New Deploy Script that supports DCOS 1.8 EE/OPEN

### PR DESCRIPTION
**Pull request by ngarleday**
I have created a new deploy script that can receive DCOS credentials from Environment variables and deploy the DB and/or Logstash containers.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -62,16 +66,19 @@ RUN rpm -ivh /tmp/epel-release-7-5.noarch.rpm \
...
          unzip \
- && pip install 'protobuf<3.0.0b1.post1' requests \
+ && pip install 'protobuf<3.0.0b1.post1' requests pexpect \
  && easy_install /tmp/*.egg \
  && pip install -r /tmp/prod_linux.txt \
  && curl -o /usr/bin/gosu -fsSL https://github.com/tianon/gosu/releases/download/1.9/gosu-amd64 \
  && chmod +sx /usr/bin/gosu \
- && rm -f /etc/httpd/conf.d/welcome.conf
+ && rm -f /etc/httpd/conf.d/welcome.conf \
+ && curl -fLsS --retry 20 -Y 100000 -y 60 https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.8/dcos -o dcos \
```
Comment (dfaller) [inline Dockerfile]: Take out the download and just move in the included binary.

File: dcos/package.json [hunk 2]
```diff
@@ -13,15 +13,16 @@
     "geospatial"
   ],
   "framework": true,
-  "version": "3.2.0",
+  "version": "3.0.0",
```
Comment (dfaller) [inline dcos/package.json]: Switch to 3.1.0

**Comment by ngarleday** [comment]
I have made updates and tested them on DREN. The updates work as expected.

**Comment by ngarleday** [comment]
Closing this pull to do a new pull from my Droessne account.

**Pull request closed without merging**